### ISSUE TITLE
Build script

### ISSUE THREAD

**Issue by ploeh** [GreanTech/Qaiain#9]
It would be nice with a command-line build script.

Usually, I do this with MSBuild, but I'm actually not too happy with that stack. I wonder if Fake would be worth looking into...

**Comment by moodmosaic** [comment]
Any thoughts on which build automations should we support? For now, I think:
- Clean build-output paths
- Compile with Release configuration
- Run unit tests with xUnit.net test runner

I think, it's worth looking into Fake mostly because of its (nice) syntax, e.g.:

```
Target "Clean" (fun _ ->
    CleanDirs ["bin"])

Target "Build" (fun _ ->
    !! ("Src/Qaiain.sln")
    |> MSBuildRelease "" "Rebuild"
    |> ignore)
```

**Comment by ploeh** [comment]
That sounds like a good starting point.

**Comment by ploeh** [comment]
Resolved by #25
